sessions/sessions.go

```diff
@@ -861,9 +863,9 @@ func (sS *SessionS) getIndexedFilters(tenant string, fltrs []string) (
 		f, err := engine.GetFilter(sS.dm, tenant, fltrID,
 			true, true, utils.NonTransactional)
 		if err != nil {
-			if err == utils.ErrNotFound {
-				err = utils.ErrPrefixNotFound(fltrID)
-			}
+			// if err == utils.ErrNotFound {
```

Comment:
Here we need to at least log it, continue will not do since there can be more serious errors coming from the database.

Reply:
Best is still to return it.